tests/acceptance/test_project_releases.py

```diff
@@ -39,11 +42,22 @@ def test_with_releases(self):
         self.browser.snapshot('project releases with releases')
 
     def test_with_no_releases(self):
+        self.dismiss_assistant()
         self.browser.get(self.path)
         self.browser.wait_until_not('.loading')
         self.browser.wait_until('.ref-project-releases')
         self.browser.snapshot('project releases without releases')
 
+    def dismiss_assistant(self):
```

Comment:
Is this something we want as a method in `AcceptanceTestCase`? Or maybe just disable entirely on the frontend?

Reply:
Disabling on the front-end would save us time from having to disable the assistant, but adds behavior that only exists in selenium which seems not great.

Adding a method to `AcceptanceTestCase` could work. We'd need a parameter for which assistant(s) need dismissal. I'm not sure I like `dismiss_assistant(2)` though.